chia/cmds/cmds_util.py

```diff
@@ -67,9 +71,12 @@ async def validate_client_connection(
             if fingerprint is None:
                 rpc_client.close()
     except ClientConnectorError:
+        rpc_client.close()
+        if not consume_errors:
+            await rpc_client.await_closed()
```

Comment:
This feels awkward, even though I can read around and figure out the pattern.  What about a `finally:` clause and just a single line calling this?  And do we want to close it for all exceptions?

Reply:
maybe i can remove this and only have the one finally

Reply:
I am going to still close it on execpt